lib/cocoapods-pod-merge/Main.rb

```diff
@@ -188,7 +188,8 @@ def parse_mergefile
     def merge(merged_framework_name, group_contents, podfile_info)
       Pod::UI.puts "Preparing to Merge: #{merged_framework_name}"
 
-      pods_to_merge = group_contents['titles']
+      # Replace the Subspecs notation (e.g AppAuth/Core) for `AppAuth` only as the folder AppAuth/Core would not exist
+      pods_to_merge = group_contents['titles'].map { |pod_name| pod_name.sub /(\/[a-zA-Z]+)/,''}.uniq
```

Comment:
Can elaborate exactly what this regex does? Will it remove any other symbols from a Pod name except `/`? Asking because I've come across pod names with symbols like `_` , `-`

Reply:
Sure the purpose of the regex is to replace subspecs like `AppAuth/Core` for `AppAuth` as the folder `AppAuth/Core` would not exist. Do you have any names of pods with the `_`, `-` in mind we can test ? Maybe I missed those cases 😞.

Reply:
Sure, for example: https://cocoapods.org/pods/Google-Mobile-Ads-SDK

Reply:
That one doesn't have sub-specs so it doesn't affect anything related to the changes. Also that one is a vendor framework so it would fail. For example I added this `youtube-ios-player-helper` to the `MergedSwiftPods` and it works successfully.

Nevertheless I detected an error that could be worth to fix later and it's related to the dashes in the module map. If I add the `youtube-ios-player-helper` to the `Subspecs` group it would fail because the `module.modulemap` doesn't like dashes in the definition of  modules

```ruby
explicit module youtube-ios-player-helper {
     header "YTPlayerView.h"
}
```

And I think that's why most of the pods using dashes they also define `module_name` (I have a PR coming soon to fix an issue related to this with `lottie-ios` as pods with `module_name` defined should use the `module_name` instead of the name of the pod specification for imports)